src/applications/ivc-champva/10-10D/components/File/FileUpload.jsx

```diff
@@ -6,7 +6,23 @@ import SchemaForm from '@department-of-veterans-affairs/platform-forms-system/Sc
 export function FileFieldCustom(props) {
   const navButtons = <FormNavButtons goBack={props.goBack} submitToContinue />;
   // eslint-disable-next-line @department-of-veterans-affairs/prefer-button-component
-  const updateButton = <button type="submit">Update page</button>;
+  const updateButton = (
+    <button type="submit" onClick={props.updatePage}>
+      Update page
+    </button>
+  );
+
+  function customSet(data) {
+    if (props.pagePerItemIndex !== undefined) {
+      // We're editing an array item (an applicant) on the review page
+      const tmpData = props.data;
+      tmpData.applicants[props.pagePerItemIndex] = data;
+      props.setFormData(tmpData);
```

Comment:
This lets us use the same file-upload custom page for items in the `applicants` list loop as well as outside.